StitchCore/StitchCoreTests/StitchCoreTests.swift

```diff
@@ -25,6 +25,48 @@ class StitchCoreTests: XCTestCase {
     }
 
 
+    func testMultipleClientStorage() throws {
+        let exp = expectation(description: "multiple clients should store separately")
+
+        let creators = [StitchClientFactory.create(appId: "test1"),
+                        StitchClientFactory.create(appId: "test2")]
+
+        when(resolved: creators).done { (results: [Result<StitchClient>]) in
+            guard case .fulfilled(let sc1) = results[0],
+                  case .fulfilled(let sc2) = results[1] else {
+                XCTFail("multiple clients could not be created")
+                exp.fulfill()
+                return
+            }
+
+            sc1.storage.set("foo", forKey: "fizz")
+            sc2.storage.set("bar", forKey: "fizz")
+
+            XCTAssertEqual(sc1.storage.value(forKey: "fizz") as? String, "foo")
+            XCTAssertEqual(sc2.storage.value(forKey: "fizz") as? String, "bar")
+            exp.fulfill()
+        }
+
+        wait(for: [exp], timeout: 10)
+    }
+
+    func testMigrationLogic() throws {
+        let exp = expectation(description: "multiple clients should store separately")
```

Comment:
[nit] the expectation description is the same as the previous test

Reply:
Good catch.